EESSI-pilot-install-software.sh

```diff
@@ -394,6 +394,34 @@ $EB --from-pr 15885 OpenBLAS-0.3.15-GCC-10.3.0.eb --robot
 $EB SciPy-bundle-2021.05-foss-2021a.eb -r --buildpath /dev/shm/$USER/easybuild_build
 check_exit_code $? "${ok_msg}" "${fail_msg}"
 
+# CUDA support
+
+# install p7zip (to be able to unpack RPMs)
+p7zip_ec="p7zip-17.04-GCCcore-10.3.0.eb"
+echo ">> Installing $p7zip_ec..."
+ok_msg="$p7zip_ec installed, off to a good (?) start!"
+fail_msg="Failed to install $p7zip_ec, woopsie..."
+$EB $p7zip_ec --robot
+check_exit_code $? "${ok_msg}" "${fail_msg}"
+
+# install CUDA (uses eb_hooks.py to only install runtime)
+cuda_ec="CUDA-11.3.1.eb"
+echo ">> Installing $cuda_ec..."
+ok_msg="$cuda_ec installed, off to a good (?) start!"
+fail_msg="Failed to install $cuda_ec, woopsie..."
+$EB $cuda_ec --robot
+check_exit_code $? "${ok_msg}" "${fail_msg}"
+
+# install CUDA samples (requires EESSI support for CUDA)
+# TODO Run EESSI NVIDIA GPU support script here
```

Comment:
We are going to need a dependent PR here, that performs the CUDA installation. We have a bit of a chicken-egg situation, as we need the installation to install the CUDA-Samples but we require the CUDA-Samples to check that things are working. This implies it'll have to be a forced installation without a verification check (which is not necessarily bad since it means we don't need a GPU).